TestCase/TestCase.m

```diff
@@ -0,0 +1,146 @@
+%This script is used to test the function. The value are compared with the
+%one obtained from: R. Tognaccini. "Lezioni di aerodinamica dell’ala rotante" 2019 pp. 84-85,fig. 7.1-4. 
+
+clear all;close all;clc;
+
+%% Import data from text file
+%% Set up the Import Options and import the data
+opts = delimitedTextImportOptions("NumVariables", 2);
+
+% Specify range and delimiter
+opts.DataLines = [1, Inf];
+opts.Delimiter = "\t";
+
+% Specify column names and types
+opts.VariableNames = ["V", "w"];
+opts.VariableTypes = ["double", "double"];
+
+% Specify file level properties
+opts.ExtraColumnsRule = "ignore";
+opts.EmptyLineRule = "read";
+
+% Import the data
+tbl = readtable("C:\Users\dtagl\Desktop\TestCase\InductionThrustCalfa0.txt", opts);
+
+%% Convert to output type
+Vt_t1 = tbl.V;
+wt_t = tbl.w;
+
+%% Clear temporary variables
+clear opts tbl
+
+%% Import data from text file
+
+
+%% Set up the Import Options and import the data
+opts = delimitedTextImportOptions("NumVariables", 2);
+
+% Specify range and delimiter
+opts.DataLines = [1, Inf];
+opts.Delimiter = "\t";
+
+% Specify column names and types
+opts.VariableNames = ["Vt", "Pt"];
+opts.VariableTypes = ["double", "double"];
+
+% Specify file level properties
+opts.ExtraColumnsRule = "ignore";
+opts.EmptyLineRule = "read";
+
+% Import the data
+tbl = readtable("C:\Users\dtagl\Desktop\TestCase\PowerThrustCalfa0.txt", opts);

```

Comment:
Do not use the global path otherwise it will not work for everyone.
Try to use `./PowerThrustCalfa0.txt` if it is in the same folder, or directly `PowerThrustCalfa0.txt`.
The same comment for the next `readable()` functions.